app/assets/js/rtMedia.backbone.js

```diff
@@ -536,6 +536,10 @@ jQuery( function( $ ) {
 				var upload_error = '';
 				var upload_error_sep = '';
 				var upload_remove_array = [ ];
+
+				var select_btn = jQuery( '.rtmedia-upload-input' );
```

Comment:
use `jQuery` or `$` consistently

Reply:
`jQuery` is mostly used in this file, that's why I used it.
Do you want me to change this entire js to either `jQuery` or `$` or should I first confirm with Pranali?

Reply:
It won't matter what you use, I recommended it for consistency.